inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/range/range_kernel_ref.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "range_kernel_ref.h"
+
+#include <kernel_selector_utils.h>
+
+namespace kernel_selector {
+
+namespace {
+
+CommonDispatchData MakeDispatchData(const range_params &params) {
```

Comment:
Not an important thing: but mostly the kernels are using the convention: SetDefault for this role. Isnt it better to unify?

Reply:
renamed locally to `SetDefault`, though it's not clear what default is being set in these functions